packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,107 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-input-focus-background-start);
+    border: var(--terra-form-input-border, 1px solid #dedfe0);
+    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    box-shadow: var(--terra-form-input-box-shadow);
+    color: var(--terra-form-input-color, #000);
+    font-family: inherit;
+    font-size: var(--terra-form-input-font-size, 1.143rem);
+    height: var(--terra-form-input-height, 2.143rem);
+    line-height: var(--terra-form-input-line-height, 1.25);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.286rem);
+    padding-left: var(--terra-form-input-padding-left, 0.286rem);
+    padding-right: var(--terra-form-input-padding-right, 0.286rem);
+    padding-top: var(--terra-form-input-padding-top, 0.286rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border: var(--terra-form-input-hover-border, 1px solid #dedfe0);
+    }
+
+    &[required] {
+      box-shadow: none;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-disabled-background-color, rgba(111, 116, 119, 0.05));
+      border: var(--terra-form-input-disabled-border, 1px solid #dedfe0);
+      color: var(--terra-form-input-disabled-color, rgba(0, 0, 0, 0.25));
+      cursor: default;
+      opacity: var(--terra-form-input-disabled-opacity);
+
+      /* stylelint-disable, selector-no-vendor-prefix */
+      // Added these in response to autoprefixer not properly adding browser imports
+      &::placeholder,
+      &:-ms-input-placeholder,
+      &::-ms-input-placeholder {
+        color: var(--terra-form-input-disabled-placeholder-color, rgba(0, 0, 0, 0.1));
+        opacity: 1;
+      }
+      /* stylelint-enable, selector-no-vendor-prefix */
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-focus-background-color, #fff);
+      background-size: var(--terra-form-input-focus-keyboard-animation-size);
+      border: var(--terra-form-input-focus-border, 1px solid #4cb2e9);
+      outline: none;
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-ltr);
+      }
+
+      [dir=rtl] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-rtl);
+      }
+    }
+
+    /* stylelint-disable, selector-no-vendor-prefix */
```

Comment:
Talked to @bjankord about an issue seen here with AutoPrefixer, this has his blessing.